packages/base-driver/lib/basedriver/helpers.js

```diff
@@ -409,6 +409,20 @@ export function generateDriverLogPrefix(obj, sessionId = null) {
   return `${obj.constructor.name}@${node.getObjectId(obj).substring(0, 4)}`;
 }
 
+/**
+ * Register the session in the context to be used in the logs
+ *
+ * @param {string} sessionId session identifier
+ */
+export function registerLogContext(sessionId) {
```

Comment:
I would rather make this method generic, so one could register any context value(s)

Reply:
I modified the function to accept an object(AppiumLoggerContext).